src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,142 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int capacity;
+    private int threshold;
+    private int size;
+    private Node<K,V> [] table;
+
+    public MyHashMap() {
+        this.table = new Node[INITIAL_CAPACITY];
+        this.capacity = INITIAL_CAPACITY;
```

Comment:
we do not need 'this' keyword here
```suggestion
        table = new Node[INITIAL_CAPACITY];
        capacity = INITIAL_CAPACITY;
```